CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Added
 - Add support for PHP 8.3 (#1218)
+- Add CSS selectors exclusion feature (#1202)
```

Comment:
This should now be #1236.
```suggestion
- Add CSS selectors exclusion feature (#1236)
```

Reply:
Yes, I thought the original that contains all discussions was more relevant.